logs_monitoring_cloudwatch_log.tf

```
@@ -9,7 +9,7 @@ resource "aws_cloudwatch_log_subscription_filter" "test_lambdafunction_logfilter
 
 resource "aws_lambda_permission" "allow_cloudwatch_logs_to_call_dd_lambda_handler" {
   count         = length(var.cloudwatch_log_groups)
-  statement_id  = "${replace(var.cloudwatch_log_groups[count.index], "/", "_")}-AllowExecutionFromCloudWatchLogs"
+  statement_id  = "${substr(replace(var.cloudwatch_log_groups[count.index], "/", "_"), 0, 60)}-AllowExecutionFromCloudWatchLogs"
```

```suggestion
  statement_id  = "${substr(replace(var.cloudwatch_log_groups[count.index], "/", "_"), 0, 67)}-AllowExecutionFromCloudWatchLogs"
```

Thank you for the contribution @jjkoh95. I think the remaining string length is 33 characters.

```terraform console
> length("-AllowExecutionFromCloudWatchLogs")
33
```

What do you think about using the remainder of the available 100 characters?

```
> length("${substr(replace("an arbitrarily long string with lots of characters and blahblahblah 104", "/", "_"), 0, 67)}-AllowExecutionFromCloudWatchLogs")
100
```

---

Hi @jim80net 
Thanks for the suggestion.
I have updated the PR.